src/client.ts

```diff
@@ -45,6 +46,7 @@ import {
 import { SpanProcessor, TestModeSpanProcessor } from "./span_processor"
 
 const DefaultInstrumentations = {
+  "@opentelemetry/instrumentation-amqp": AmqplibInstrumentation,
```

Comment:
Let's make this match the name of the actual instrumentation package.

```suggestion
  "@opentelemetry/instrumentation-amqplib": AmqplibInstrumentation,
```